docs/user-tutorial/result-summary.md

```diff
@@ -14,7 +14,7 @@ This tool is to generate a readable summary report based on the raw benchmark re
 
 2. Prepare the raw data and rule file on the local machine.
 
-3. Generate the result summary automatically using `sb result summary` command. The detailed command can be found from [SuperBench CLI](../cli).
+3. Generate the result summary automatically using `sb result summary` command. The detailed command can be found from [SuperBench CLI](../cli.md).
```

Comment:
Hi, our doc is in website [Result Summary | SuperBench](https://microsoft.github.io/superbenchmark/docs/user-tutorial/result-summary.)
I think the original one works well in my side, and your patch seems can not work, could you please help me check that?

Reply:
Hi there, the link: [Result Summary | SuperBench](https://microsoft.github.io/superbenchmark/docs/user-tutorial/result-summary.) appears broken on my side... But may just include a trailing period.  I think I see what's happing: I pushed an edit that fixed a link as viewed _within the markdown_ documentation on github. You're absolutely, this functions fine on the website. Apologies for the confusion.

Reply:
Hi @yukirora, according to [docusaurus](https://docusaurus.io/docs/docs-markdown-features#referencing-other-documents), it seems relative file paths (with `.md` or `.mdx`) is preferred than relative URL links. Both will work for website and relative file paths also work on GitHub. Maybe I can merge @jaredAMD's PR first and you can update others paths under `docs/` when you have time?

> I think the original one works well in my side, and your patch seems can not work

It should work fine because docs-build check will check all links? Both `../cli` and `../cli.md` should work

Reply:
ok, thanks @abuccts, If so, you can merge this PR first